src/resonanceReconstruction/rmatrix/legacy/resolved/Resonance.hpp

```diff
@@ -69,6 +69,11 @@ class Resonance {
    */
   const Width& fission() const { return this->fission_; }
 
+  /**
+   *  @brief Return whether or not the resonance has fission or not
+   */
+  bool hasFission() const { return this->fission_.value != 0.0; }
```

Comment:
Is the "resolved" namespace just BW resonances?  Because RM would have multiple fission channels.  This looks to be templated, though -- am I just not seeing the specialization yet?

Reply:
No. legacy::resolved::Resonance is not templated. It is a concrete type. You may be confusing it with the SpinGroup, which is templated on MLBW and SLBW.